docs/README.md

```diff
@@ -18,11 +18,17 @@ Set up and configure the main application.
 
 ## Luigi Client
 
-Use the Luigi Client API documentation to learn more about the functions and parameters you can use to communicate with the core application to define lifecycle, links, and the appearance of the application.
+Once you have configured Luigi Core, you can look at the Luigi Client API documentation. Luigi Client enables communication between your main application and a micro frontend. Learn more about the functions and parameters you can use to define lifecycles, links, and the appearance of the application.
 
+* [Communication](communication.md) describes how to configure features related to communication between Luigi Core and Client.
 * [Lifecycle](luigi-client-api.md#lifecycle) gives you details on the lifecycle of listeners, navigation nodes, and Event data.
 * [Link Manager](luigi-client-api.md#linkmanager) allows you to navigate to a specific route. 
-* [UX Manager](luigi-client-api.md#uxmanager) helps you to manage the appearance features in Luigi, such as the behavior of backdrop or loading indictors.
+* [UX Manager](luigi-client-api.md#uxmanager) helps you manage the appearance features in Luigi, such as the behavior of backdrop or loading indictors.
 
-## Common
-* [Communication](communication.md) describes how to configure features related to communication between Luigi Core and Client.
+## Examples
+
+Follow the Luigi [application examples](core/examples) for an in-depth look at Luigi capabilities. 
```

Comment:
```suggestion
Follow the Luigi [application examples](../core/examples/README.md) for an in-depth look at Luigi capabilities. 
```
This link won't work, because examples are not in the `docs` directory - it worked in the main README, as `core/examples` is located in the main Luigi directory. To make it work, you can either use the absolute path (https://github.com/SAP/luigi/tree/master/core/examples) or a relative one `(../core/examples)` to go one level up and then point to the right directory. :)
I pointed to the readme.md file directly :) 
To check the links, the best way is to view this file as rendered Markdown (using the `...` > View File in the right top corner (under Add your review). Of course, sometimes they won't work - if you are, for example, linking to a doc which you add in the same PR - but if the doc exists already it should be easy to check if the link is not broken.